graphql-cli/src/main/java/io/ballerina/graphql/generator/ballerina/FunctionBodyGenerator.java

```diff
@@ -173,18 +177,72 @@ public FunctionBodyNode generateInitFunctionBody(AuthConfig authConfig) {
      * @param authConfig the object instance representing authentication configuration information
      * @return           the object instance representing http client config
      */
-    private StatementNode createHttpClientConfig(AuthConfig authConfig) {
-        String auth = "";
+    private List<StatementNode> generateHttpClientConfigurationNode(AuthConfig authConfig) {
```

Comment:
Let's add a sample code generated under the documentation comment.

Reply:
Added in https://github.com/ballerina-platform/graphql-tools/commit/820d69eccf3d162405d31e4797434fb404c1da4e